### ISSUE TITLE
"Choose max amount" not working for "Add sell level"

### ISSUE THREAD

**Issue by offtopic** [MarcelBeining/EazeBot#14]
**Describe the bug**
Creating a new trade set with adding a new sell level with a specific price.
After that you enter the amount to sell or "choose max amount"
Entering an amount works but on clicking "choose max amount" an error is thrown:

`2020-08-04 19:57:05,369 [Bot:73674776] [INFO ] Finished updating trade sets...
No error handlers are registered, logging exception.
Traceback (most recent call last):
File "/usr/local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 340, in process_update
handler.handle_update(update, self, check, context)
File "/usr/local/lib/python3.8/site-packages/telegram/ext/conversationhandler.py", line 465, in handle_update
new_state = handler.handle_update(update, dispatcher, check_result, context)
File "/usr/local/lib/python3.8/site-packages/telegram/ext/handler.py", line 119, in handle_update
return self.callback(update, context)
File "/eazebot_docker/eazebot/bot.py", line 1012, in inline_button_callback
return context.user_data['lastFct'].pop()(float(args.pop(0)))
File "/eazebot_docker/eazebot/bot.py", line 659, in <lambda>
user_data['lastFct'].append(lambda r: self.ask_pos(context, exch, uid_ts, direction, 'amount', r, utid))
File "/eazebot_docker/eazebot/bot.py", line 665, in ask_pos
response = response / utid.price
AttributeError: 'str' object has no attribute 'price'`

 - OS: Windows 10 64 bit
 - Version eazebot as shown in botInfo 2.6.0 (docker version)
 - Version of ccxt: 1.32.12

Its minor.

**Comment by offtopic** [comment]
Okay, after another try it seems to be an issue if the price is entered based on the toggled currency:
In my case I added a new tradeset for xmr/btc and if you toogle btc it generates the error on entering the sell amount in BTC.

```
No error handlers are registered, logging exception.

Traceback (most recent call last):

File "/usr/local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 340, in process_update

handler.handle_update(update, self, check, context)

File "/usr/local/lib/python3.8/site-packages/telegram/ext/conversationhandler.py", line 465, in handle_update

new_state = handler.handle_update(update, dispatcher, check_result, context)

File "/usr/local/lib/python3.8/site-packages/telegram/ext/handler.py", line 119, in handle_update

return self.callback(update, context)

File "/eazebot_docker/eazebot/bot.py", line 122, in received_float

return context.user_data['lastFct'].pop()(float(update.message.text))

File "/eazebot_docker/eazebot/bot.py", line 659, in <lambda>

user_data['lastFct'].append(lambda r: self.ask_pos(context, exch, uid_ts, direction, 'amount', r, utid))

File "/eazebot_docker/eazebot/bot.py", line 665, in ask_pos

response = response / utid.price

AttributeError: 'str' object has no attribute 'price'

```

it is working if I enter the amount in XMR

**Comment by MarcelBeining** [comment]
Closed with [v2.6.1](https://github.com/MarcelBeining/EazeBot/releases/tag/v2.6.1)
